File: Module 6 - Python/mario/more/mario.py
from cs50 import get_int

while True:
    size = get_int("Height: ")
    if 0 < size < 9:
        break

for i in range(size):
    air = size - i - 1

    for x in range(air):
        print(" ", end="")

    for x in range(i + 1):
        print("#", end="")

    print("  ", end="")

    for x in range(i + 1):
        print("#", end="")

    print()

File: Módulo 2 - C/Mario (Easy)/mario.c
#include <stdio.h>
//#include <cs50.h>

int main(void)
{
    int size;

    do
    {
        //Get a number of User
        size = get_int("Enter tower size: ");
    }
    while(size < 1 || size > 8);

    // Loop responsible for drawing the lines
    for(int i = 0; i < size; i++)
    {
        // Number of empty spaces a line will have
        int spacesCount = size - (i + 1);

        // Loop responsible for placing the characters on each line
        for(int x = 0; x < size; x++)
        {
            if(spacesCount > 0)
            {
                printf(" ");
                spacesCount--;
            }
            else
            {
                printf("#");
            }
        }

        printf("\n");
    }
}